internal/cmd/invoice_list.go

```diff
@@ -46,25 +55,29 @@ func printInvoiceListTable(invoices []turso.Invoice) {
 	printTable(headers, data)
 }
 
-func printInvoiceListPdfs(invoices []turso.Invoice) {
-	headers, data := invoiceListPdfs(invoices)
+func printInvoiceListLinks(invoices []turso.Invoice) {
+	headers, data := invoiceListLinks(invoices)
 	printTable(headers, data)
 }
 
 func invoiceListTable(invoices []turso.Invoice) ([]string, [][]string) {
-	headers := []string{"ID", "Amount Due", "Due Date", "Paid At", "Payment Failed At"}
+	headers := []string{"ID", "Amount Due", "Status", "Due Date", "Paid At", "Payment Failed At"}
 	data := make([][]string, len(invoices))
 	for i, invoice := range invoices {
-		data[i] = []string{invoice.Number, invoice.Amount, invoice.DueDate, invoice.PaidAt, invoice.PaymentFailedAt}
+		data[i] = []string{invoice.Number, invoice.Amount, invoice.Status, invoice.DueDate, invoice.PaidAt, invoice.PaymentFailedAt}
 	}
 	return headers, data
 }
 
-func invoiceListPdfs(invoices []turso.Invoice) ([]string, [][]string) {
+func invoiceListLinks(invoices []turso.Invoice) ([]string, [][]string) {
```

Comment:
⛏️⛏️⛏️: `s/invoiceList/invoice/g` on function names. Same thing for `invoiceType` below. Feel free to ignore.